docs/ManualTests.md

```diff
@@ -237,11 +237,11 @@ Verify the following:
 For Programmer Mode
 Verify the following:
 6.	"Bit Toggling Keypad":
-*Expected: In app keypad changes to represent Bits (1s and 0s).*
+*Expected: In-app keypad changes to represent Bits (1s and 0s).*
 7.	"QWORD / DWORD / WORD / BYTE":
 *Expected: Toggles as expected.*
 8.	"Hex" Hexadecimal:
-*Expected: A B C D E F become active and user can use them. A maximum of 16 characters can be entered.*
+*Expected: A B C D E F become active and the user can use them. A maximum of 16 characters can be entered.*
```

Comment:
-0, [the] is implied and it's ok in semi-informal English to omit

Reply:
"and user can use" vs. "and the user can use"... I think that adding "the" makes it read better, especially with "user" and "use" being so close together in the same sentence.